tests/functional/test_docker.py

```diff
@@ -1600,3 +1600,10 @@ def test_version_bare(chalk_default: Chalk):
         params=["version"],
     )
     assert run
+
+
+def test_piped_dockerfile_has_no_relative_path(chalk: Chalk, random_hex: str):
+    tag_base = f"{REGISTRY}/nonchalked_{random_hex}"
+    tag = f"{tag_base}:latest"
+    _, result = chalk.docker_build(content="FROM alpine", tag=tag)
+    assert "DOCKERFILE_PATH_WITHIN_VCTL" not in result.mark
```

Comment:
this is fairly expensive test as it needs to do docker build, then chalked docker build just to assert a single field in the report. we already have other tests which pass dockerfile via stdin. can we adjust them to do this assertion as well. prolly will be something like:

```python
assert build.mark.has(
  ...
  DOCKERFILE_PATH_WITHIN_VCTL=MISSING,
)
```

Reply:
moved assertion under `test_base_images`